### ISSUE TITLE
IX subnet changed with peers still attached to previous subnet

### ISSUE THREAD

**Issue by DamianZaremba** [peeringdb/peeringdb#638]
I'm not really sure this is an issue, more than an implementation/process/expectation best practice clarification.

Equinix IX Miami (https://www.peeringdb.com/ix/17) had a subnet changed (`198.32.124.0/23` -> `198.32.242.0/23`, `2001:478:124::/64` -> `2001:504:0:6::/64`), after which there are peers in the new and old subnets;

'old':
```
Fastly, Inc.
54113
198.32.125.198
2001:478:124::1198
30G
Selective
```

'new':
```
Aptum Technologies
13768
198.32.242.125
2001:504:0:6:0:1:3768:1
10G
Selective
```

When trying to update an existing netixlan the validation appropriately fails due to one of the addresses (actually both) not being within the ixlan subnets.

This was noticed by a quirk on my end where I update the ips separately to avoid creating separate v4 and v6 entries and/or removing re/adding entries when a single address changes.

Naturally while a subnet change is under way it's correct to have peers in two different subnets as they turn up the second range, until they turn down the first range.... however I expected the process in PeeringDB to go something like;

* New subnet is added to the IX (second ixlan?)
* Peers can create attachments in the second range
* IX decommissions the old range
* 'old' subnet is deleted in PeeringDB, removing all associated peers

Similar to what I assume happens in the case that an IX goes out of business.

In this case the sequence of events appears to have been;
* Subnet was changed
* New peer attachments made under new subnet
* Old attachments now fail validation for updates and are effectively orphaned until the network cleans them up

For operators using the relationship data for automation, being unable to manage the previous attachment during the migration is a little tricky.

**Comment by arnoldnipper** [comment]
It's up to the `ix` how to do a migration. Ideally, they add the 2nd `ixpfx` (not `ixlan`) and when the migration period is over they ask us to batch migrate the left-overs. 

So I don't really get where your issue is.

**Comment by arnoldnipper** [comment]
@DamianZaremba this issue is fixed by #696
